README.md

```diff
@@ -46,10 +46,14 @@ Encode an ipld object as a CID that uses the identity hash.
 ### decodeIdentityCID(cid: CID): Record<string, any>
 Decode an ipld object from a CID that uses the identity hash.
 
+### toJWSPayload(payload: EncodedPayload | CID): string
+Transform an `EnvodedPayload` (from `encodePayload()`) or a CID into a JWS string for use with `createJWS()` in [did-jwt](https://github.com/decentralized-identity/did-jwt). The string form is simply the Base64url encoded form of the CID's byte representation.
+
+### toJWSStrings(jose: any): string[]
```

Comment:
Unfortunate that you need this method. Ideally `did-jwt` should support the General encoding, that the ´jose` object here has, out of the box!